### ISSUE TITLE
Accessibility Audit: Heading level 1 not available under Listing Detail pages

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#950]
Turn on the Screen Reader and go to a Listing Detail page page. Note that the page doesn't have a heading Level 1. This page should have a heading level 1 and headings in the page should be in a hierarchical order.

https://user-images.githubusercontent.com/394908/148309292-b168718b-9ed9-47aa-9711-2d17391ac0bc.mov

**Comment by emilyjablonski** [comment]
Fixed in Seeds
